Program.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HW2
+{
+    class Translit
+    {
+        static void Main(string[] args) //Display the translated string
+        {
+            Translit translit = new Translit(); //Create example of class Translit
+            string myString;
+            Console.WriteLine("Enter a string: ");
+            myString = Console.ReadLine();
+            Console.WriteLine(translit.translitToOtherLanguage(myString)); 
+        }
+        private string translitToOtherLanguage(string str)//This method translates the text of one language to another 
+        {
+            StringBuilder retranslit = new StringBuilder ();

```

Comment:
use create massive any time you use this method, better to initialize it once in a class constructor